packages/atlas/src/components/_inputs/Input/Input.tsx

```diff
@@ -61,6 +63,24 @@ const InputComponent: React.ForwardRefRenderFunction<HTMLInputElement, InputProp
 
   const inputRef = useRef<HTMLInputElement>(null)
 
+  const isNodeStartText = nodeStart && typeof nodeStart === 'string'
+  const isNodeEndText = nodeEnd && typeof nodeEnd === 'string'
+
+  const renderNodeStart = isNodeStartText ? (
+    <Text variant={size === 'large' ? 't300' : 't200'} color={cVar('colorTextMuted')}>
+      {nodeStart}
+    </Text>
+  ) : (
+    nodeStart
+  )
+  const renderNodeEnd = isNodeEndText ? (
+    <Text variant={size === 'large' ? 't300' : 't200'} color={cVar('colorTextMuted')}>
+      {nodeEnd}
+    </Text>
+  ) : (
+    nodeEnd
+  )
```

Comment:
Let's rename those vars - `render*` suggests it's a function that returns JSX while it's a static JSX instead. Maybe `rendered`?